plugins/inputs/nvidia_smi/nvidia_smi.go

```diff
@@ -168,7 +171,7 @@ func setIfUsed(t string, m map[string]interface{}, k, v string) {
 			}
 		}
 	case "int":
-		if val != "" {
+		if val != "" && val != "N/A" {
```

Comment:
Are you currently seeing metrics come in with "N/A"? My only concern is if users are expecting "N/A" and then they suddenly stop seeing them.

In general though I would be fine with this.

Reply:
This is for `int` conversion only. `N/A` will be equal to the absence of value. I see these values when the value does not make sense. For example in A10 I can see (in real world `nvidia-smi` output):
```xml
...
        <retired_pages>
            <multiple_single_bit_retirement>
                <retired_count>N/A</retired_count>
...
```
because A10 does not support page retires. For whatever reason xml still contains tags.